xarray/backends/zarr.py

```diff
@@ -395,6 +405,14 @@ def open_zarr(store, group=None, synchronizer=None, auto_chunk=True,
     http://zarr.readthedocs.io/
     """
 
+    if auto_chunk and chunks is None:
+        chunks = 'auto'  # maintain backwards compatibility
```

Comment:
Add a FutureWarning here saying that `auto_chunk` is deprecated and should be replaced by `chunks`. This will need an appropriate test too.